docs/user-guide/generate-certificates.md

```diff
@@ -102,21 +91,79 @@ Because `--update-config` was specified the `zowe.certificates` section's values
       certificateAuthorities: /global/zowe/keystore/local_ca/local_ca.cer
 ```
 
-When using a self-signed certificate, you will be challenged by your browser when logging in to Zowe to accept its untrusted certificate authority.  Depending on the browser you are using there are different ways to proceed.  
+When using a self-signed certificate, you will be challenged by your browser when logging in to Zowe to accept its untrusted certificate authority.  Depending on the browser you are using there are different ways to proceed.
 
-### Manually import a certificate authority into a web browser
+## Create a self signed JCERACFKS certificate
 
-To avoid the browser untrusted CA challenge, you can import Zowe's certificates into the browser to avoid untrusted network traffic challenges. For more information, see [Import the local CA certificate to your browser](../extend/extend-apiml/certificate-management-in-zowe-apiml.md/#import-the-local-ca-certificate-to-your-browser).
+The following `zowe.yaml` example will generate:
 
-To avoid requiring each browser to trust the CA that signed the Zowe certificate, you can use a public certificate authority such as _Symantec_, _Comodo_, _Let's Encrypt_, or _GoDaddy_to create a certificate. These certificates are trusted by all browsers and most REST API clients. This option, however, requires a manual process to request a certificate and may incur a cost payable to the publicly trusted CA.
+ - A `JCERACFKS` certificate, specified in `zowe.setup.certificate.type` 
+ - A keyring named `ZoweKeyring` specified in  `zowe.setup.certificate.keyring.name`. 
+ - A certificate with the label `localhost` specified in `zowe.setup.certificate.keyring.label`  
+ - A certificate authority with the label `localca` specified in  `zowe.setup.certificate.keyring.caLabel` with a common name `Zowe Service CA`.
```

Comment:
Try to standardize your style here. Decide whether to use a comma before "specified" (as in your first bullet). Also decide whether to end each bullet w/ a period or not.